README.md

```diff
@@ -33,16 +33,16 @@ If you wish to contribute to this template, the following information may be hel
  * Android SDK : 22+ or newer
 
 ## Build instructions
- * Edit [fhconfig.properties](app/src/main/assets/fhconfig.properties) to include the relevant information from RHMAP.  
+ * Edit `fhconfig.properties` to include the relevant information from RHMAP.  
  * Attach running Android Device with API 16+ running  
- * Edit [FHOauth](app/src/main/java/com/feedhenry/oauth/oauth_android_app/FHOAuth.java) and set "FH_AUTH_POLICY" to your auth policy if you did not name yoru policy "Google".
+ * Edit `FHOAuth.java` and set "FH_AUTH_POLICY" to your auth policy if you did not name yoru policy "Google".
  * ./gradlew installDebug  
  
 ## How does it work?
 
 ### Checking if a user is logged in
 
-After having [initialized](app/src/main/java/com/feedhenry/oauth/oauth_android_app/FHOAuth.java#L48) the SDK, you may use `FHAuthSession` to check a user's logged in status.  `FHAuthSession.getToken()` will display a user's cached token and `FHAuthSession.verify` can be used to ensure the session is still valid on the server.
+After having `FHOAuth.java` (line 48) the SDK, you may use `FHAuthSession` to check a user's logged in status.  `FHAuthSession.getToken()` will display a user's cached token and `FHAuthSession.verify` can be used to ensure the session is still valid on the server.
```

Comment:
Let's avoid to add _line_  on this because it change from time to time and we always forget to change it on README ;)